### PR TITLE
rename ossec.conf name to manager or agent

### LINKED ISSUE #955: Rename ossec.conf to manager.conf and agent.conf in Splunk
The purpose of this issue is to rename `ossec.conf` in Wazuh Splunk according to https://github.com/wazuh/wazuh/issues/7787:

- **Agent configuration**: All settings related to Wazuh Agent will be located at `agent.conf`. Blocks:
  - `<active-response>` (client side)
  - `<agent-upgrade>` (client side)
  - `<client>`
  - `<client_buffer>`
  - `<fluent-forward>`
  - `<gcp-pubsub>`
  - `<labels>` (same as manager)
  - `<localfile>`
  - `<logging>` (same as manager)
  - `<rootcheck>`
  - `<sca>`
  - `<socket>`
  - `<syscheck>`
  - `<wodle>`
    - `<wodle name="agent-key-polling">`
    - `<wodle name="aws-s3">`
    - `<wodle name="azure-logs">`
    - `<wodle name="cis-cat">`
    - `<wodle name="command">`
    - `<wodle name="docker-listener">`
    - `<wodle name="open-scap">`
    - `<wodle name="osquery">`
    - `<wodle name="syscollector">`

- **Manager configuration**: All settings related to Wazuh Manager will be located at `manager.conf`. Blocks:
  - `<active-response>` (manager side)
  - `<agentless>`
  - `<agent-upgrade>` (manager side)
  - `<alerts>`
  - `<auth>`
  - `<cluster>`
  - `<command>`
  - `<database_output>`
  - `<email_alerts>`
  - `<global>`
  - `<integration>`
  - `<labels>` (same as agent)
  - `<logging>` (same as agent)
  - `<remote>`
  - `<reports>`
  - `<rule_test>`
  - `<ruleset>`
  - `<syslog_output>`
  - `<task-manager>`
  - `<vulnerability-detector>`

Also, to avoid confusion, `agent.conf` shared configuration file should be renamed to `shared.conf` and `agent-template.conf` file to `shared-template.conf`.

In addition, `gen_ossec.sh` script file has to be renamed to `gen_wazuh.sh`.

### PR DESCRIPTION
**Pull request by sortizowlh**
Hi team!
This PR change the references to ossec.conf file
Closes #955

### CODE REVIEW

File: SplunkAppForWazuh/default/app.conf [hunk 1]
```diff
@@ -3,7 +3,7 @@ is_visible = 1
 label = Wazuh
 
 [launcher]
-version = 4.0.4
+version = 4.1.3
```
Comment (CPAlejandro) [inline SplunkAppForWazuh/default/app.conf:6]: I think these changes should not be included in your pr

**Review by asteriscos — APPROVED** [review]
LGTM!

File: SplunkAppForWazuh/default/app.conf (hunk 1 above)
Comment (sortizowlh) [reply SplunkAppForWazuh/default/app.conf]: Changed version!

**Review by TomasTurina — COMMENTED** [review]
We have also decided to rename `agent.conf` shared configuration file to `shared.conf`, to avoid confusion with names. Please, take a look at this and rename them if applies.

**Pull request merged**